nvm.psm1

```diff
@@ -87,7 +87,7 @@ function Set-NodeVersion {
 
     $Version = $Version.Trim()
 
-    $matchedVersion = if (!($Version -match "v\d\.\d{1,2}\.\d{1,2}")) {
+    $matchedVersion = if (!($Version -match "v\d*\.\d*\.\d*")) {
```

Comment:
`*` would allow no digits too, e.g. `v...` or `v1.2.` or `v..2`. I think this should be `+`

Reply:
Oh yeah... my regex-foo failed.

Pushed a proper regex